sqlmesh/core/model/common.py

```diff
@@ -109,6 +109,43 @@ def parse_properties(cls: t.Type, v: t.Any, values: t.Dict[str, t.Any]) -> t.Opt
     return properties
 
 
+def default_catalog(cls: t.Type, v: t.Any) -> t.Optional[str]:
+    if v is None:
+        return None
+    if isinstance(v, exp.Expression):
+        return v.meta.get("sql") or v.sql()
+    return str(v)
+
+
+@field_validator_v1_args
+def depends_on(cls: t.Type, v: t.Any, values: t.Dict[str, t.Any]) -> t.Optional[t.Set[str]]:
+    dialect = values.get("dialect")
+    default_catalog = values.get("default_catalog")
+
+    if isinstance(v, (exp.Array, exp.Tuple)):
+        return {
+            normalize_model_name(
+                table.name if table.is_string else table.sql(dialect=dialect),
```

Comment:
why are you calling table.sql() here, just pass in the table/column, normalize_model_name takes str, table, and column, calling sql() is a waste

Reply:
the signature of the function doesn't change, it already accepts both

Reply:
@eakmanrq this was not addressed, you don't need to call table.sql(), just pass in table